### DISCUSSION TITLE
/cn/2019/04/heuristic-algorithms/

### DISCUSSION THREAD

**Discussion by leovan** [leovan/leovan.me#11] [Comments]
启发式算法 (Heuristic Algorithms) 
 https://leovan.me/cn/2019/04/heuristic-algorithms/

**Comment by leovan** [comment]
```header
由 Osinovsky 于 2020-08-03 01:57:22 写道：
````
感谢分享！您的博客是我见过最漂亮的ヽ(✿ﾟ▽ﾟ)ノ能透露一下建站的框架吗？

**Reply by leovan** [reply]
```header
于 2020-08-03 02:35:43 写道：
````
谢谢。网站是利用 Hugo 搭建的，我用了 R 里面封装的一个上层包 blogdown
，当然这个是非必须的。主题最早也是抄的谢益辉大神的博客：[https://yihui.org/](https://yihui.org/)。后面我有做了很多的改动，整个博客是开源的：[https://github.com/leovan/leovan.me](https://github.com/leovan/leovan.me)，您有兴趣的话可以研究一下。

**Comment by Wxy985644020** [comment]
看到您的简历，十分羡慕且敬佩，非常感谢您的分享！！

**Comment by pursuitxi** [comment]
Rosenbrock 函数 是在(1,1)取到最优点吧（?）

**Reply by leovan** [reply]
是的，笔误，已修正。

**Comment by ChrisShenX** [comment]
超启发式算法 使用的什么绘图工具？

**Reply by leovan** [reply]
那个是截图，用 draw.io 之类的应该都可以画出来。
